packages/xstate-graph/test/index.test.ts

```diff
@@ -413,33 +413,33 @@ describe('state tests', () => {
   });
 
   it('should test with input', () => {
-    const model = createTestModel(
```

Comment:
This was a little weird too... for some reason, TS complained if `setup(…)` was inside `createTestModel(setup(…).createMachine(…))` but not if I moved it outside.